cl/search/views.py

```diff
@@ -488,7 +493,13 @@ def show_results(request: HttpRequest) -> HttpResponse:
                         "value"
                     ] = render_dict["search_summary_str"]
                     render_dict.update({"alert_form": alert_form})
-
+                elif request.GET.get("type") == SEARCH_TYPES.PEOPLE:
```

Comment:
Can we factor out `request.GET.get("type")` at the top or something, please?

Reply:
sure!